File: src/main/java/entites/Transport.java
package entites;

import java.time.LocalDate;

public class Transport {
    public String getNum_t;
    private int num_t;
    private int num_ch;
    //transpoteur transpoteur;
    private LocalDate dd;
    private LocalDate da;
    private int num_v;
   // vehicule vehicule;
    private int cout;

    public Transport() {
    }

    public  Transport(int num_ch, LocalDate dd, LocalDate da, int num_v, int cout) {
        this.num_t= num_t;
        this.num_ch = num_ch;
        this.dd = dd;
        this.da = da;
        this.num_v = num_v;
        this.cout = cout;
    }

    public Transport(int numCh, int numV, int cout, LocalDate dd, LocalDate da) {
    }





   /* public Transport(int i, int i1, LocalDate of, LocalDate of1, int i2, int i3) {
    }*/

    /*public transport(String nom, String prenom) {
        this.nom = nom;
        this.prenom = prenom;
    }*/

    /*public transport(int num_ch, entites.transpoteur transpoteur, LocalDate dd, LocalDate da, int num_v, entit int cout) {
        this.num_ch = num_ch;
        //this.transpoteur = transpoteur;
        this.dd = dd;
        this.da = da;
        this.num_v = num_v;
        //this.vehicule = vehicule;
        this.cout = cout;
    }*/

    /*public entites.transpoteur getTranspoteur() {
        return transpoteur;
    }*/

    /*public void setTranspoteur(entites.transpoteur transpoteur) {
        this.transpoteur = transpoteur;
    }*/

    public LocalDate getDd() {
        return dd;
    }

    public void setDd(LocalDate dd) {
        this.dd = dd;
    }

    public LocalDate getDa() {
        return da;
    }

    public void setDa(LocalDate da) {
        this.da = da;
    }

    /*public entites.vehicule getVehicule() {
        return vehicule;
    }*/

    /*public void setVehicule(entites.vehicule vehicule) {
        this.vehicule = vehicule;
    }*/

    public int getNum_t() {
        return num_t;
    }

    public void setNum_t(int num_t) {
        this.num_t = num_t;
    }

    public int getNum_ch() {
        return num_ch;
    }

    public void setNum_ch(int num_ch) {
        this.num_ch = num_ch;
    }




    public int getNum_v() {
        return num_v;
    }

    public void setNum_v(int num_v) {
        this.num_v = num_v;
    }

    public int getCout() {
        return cout;
    }

    public void setCout(int cout) {
        this.cout = cout;
    }

    @Override
    public String toString() {
        return "transport{" +
                "num_t=" + num_t +
                ", num_ch=" + num_ch +
                ", dd=" + dd +
                ", da=" + da +
                ", num_v=" + num_v +
                ", cout=" + cout +
                '}';
    }
}

File: Transport.php
<?php

namespace App\Entity;

use Doctrine\DBAL\Types\Types;
use Doctrine\ORM\Mapping as ORM;

/**
 * Transport
 *
 * @ORM\Table(name="transport", indexes={@ORM\Index(name="num_ch", columns={"num_ch"}), @ORM\Index(name="num_v", columns={"num_v"})})
 * @ORM\Entity
 */
class Transport
{
    /**
     * @var int
     *
     * @ORM\Column(name="num_t", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $numT;

    /**
     * @var int
     *
     * @ORM\Column(name="num_ch", type="integer", nullable=false)
     */
    private $numCh;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="dd", type="date", nullable=false)
     */
    private $dd;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="da", type="date", nullable=false)
     */
    private $da;

    /**
     * @var int
     *
     * @ORM\Column(name="num_v", type="integer", nullable=false)
     */
    private $numV;

    /**
     * @var int
     *
     * @ORM\Column(name="cout", type="integer", nullable=false)
     */
    private $cout;

    public function getNumT(): ?int
    {
        return $this->numT;
    }

    public function getNumCh(): ?int
    {
        return $this->numCh;
    }

    public function setNumCh(int $numCh): static
    {
        $this->numCh = $numCh;

        return $this;
    }

    public function getDd(): ?\DateTimeInterface
    {
        return $this->dd;
    }

    public function setDd(\DateTimeInterface $dd): static
    {
        $this->dd = $dd;

        return $this;
    }

    public function getDa(): ?\DateTimeInterface
    {
        return $this->da;
    }

    public function setDa(\DateTimeInterface $da): static
    {
        $this->da = $da;

        return $this;
    }

    public function getNumV(): ?int
    {
        return $this->numV;
    }

    public function setNumV(int $numV): static
    {
        $this->numV = $numV;

        return $this;
    }

    public function getCout(): ?int
    {
        return $this->cout;
    }

    public function setCout(int $cout): static
    {
        $this->cout = $cout;

        return $this;
    }


}
